.github/workflows/npm.yml

```diff
@@ -56,7 +58,7 @@ jobs:
 
     - run: npx prebuildify --napi -t ${{ steps.get-nvm-version.outputs.node-version }} --arch arm64
       working-directory: node
-      if: matrix.os == 'macos-11'
+      if: matrix.os != 'ubuntu-latest'
```

Comment:
Let's make this a little more semantic: we can rename `additional-rust-target` to `arm64-rust-target` and then this can check whether `arm64-rust-target` is present at all.

Reply:
Makes sense! Updated in https://github.com/signalapp/libsignal-client/pull/440/commits/a2b1ae1d960076fbcf46b576d2f038fa5d2ac53d - seems to work correctly: https://github.com/dennisameling/libsignal-client/runs/4893965003?check_suite_focus=true